### DISCUSSION TITLE
Can't start the server

### DISCUSSION THREAD

**Discussion by billychannnnnn** [thijsvanloef/palworld-server-docker#125] [Q&A]
```
*****CHECKING FOR EXISTING CONFIG*****
*****GENERATING CONFIG*****
[S_API] SteamAPI_Init(): Loaded local 'steamclient.so' OK.
CAppInfoCacheReadFromDiskThread took 14 milliseconds to initialize
Setting breakpad minidump AppID = 2394010
[S_API FAIL] Tried to access Steam interface SteamUser021 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface SteamFriends017 before SteamAPI_Init succeeded.
Signal 11 caught.
RCON_ENABLED=true
RCON_PORT=25575
*****STARTING SERVER*****
./PalServer.sh -port=8211 -players=16 -serverpassword="worldofpals" -adminpassword="xxxxxx" -queryport=27015 -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS
```

docker logs stuck in here and i can't connect to the server in game.  What should i do?

**Comment by thijsvanloef** [comment]
This is very strange behavior indeed. Could you try using the latest update of the Image? 

- `docker compose down --rmi all`
- `docker compose up -d`

and see if the issue persist?

**Reply by billychannnnnn** [reply]
thx for ur reply.

I just found out the container can use only 1 core cpu (there are 3 more idled) for unknown reason and somehow a server reboot fix it… proper terminal prompts were showed and successfully connect to the server afterwards.

It seems to be the problem of my server itself, not this repo.
